src/app/shared/components/template/components/task-card/task-card.component.scss

```diff
@@ -103,44 +103,111 @@
       padding-left: var(--small-padding);
     }
   }
+}
 
-  .badge {
+.badge {
+  position: absolute;
+  top: -10px;
+  filter: drop-shadow(var(--ion-default-box-shadow));
+  &.highlighted-badge {
+    right: -10px;
+    padding: 5px 10px;
+    border-radius: var(--ion-border-radius-small);
+    background: var(--ion-color-secondary);
+    color: white;
+    font-weight: var(--font-weight-bold);
+  }
+  &.progress-badge {
+    right: -12px;
+    width: 36px;
+  }
+  .circle {
+    height: 36px;
+    width: 36px;
+    border-radius: 50%;
     position: absolute;
-    top: -10px;
-    filter: drop-shadow(var(--ion-default-box-shadow));
-    &.highlighted-badge {
-      right: -10px;
-      padding: 5px 10px;
-      border-radius: var(--ion-border-radius-small);
-      background: var(--ion-color-secondary);
-      color: white;
-      font-weight: var(--font-weight-bold);
+    z-index: 1;
+    &.completed {
+      background-color: var(--ion-color-green);
     }
-    &.progress-badge {
-      right: -12px;
-      width: 36px;
+    &.inProgress {
+      background-color: var(--ion-color-gray-light);
     }
-    .circle {
-      height: 36px;
-      width: 36px;
-      border-radius: 50%;
+  }
+  .icon {
+    position: absolute;
+    z-index: 2;
+    width: 100%;
+    padding: 4px;
+    &.completed {
+      top: 2px;
+      padding: 8px;
+    }
+  }
+}
+
+.circle-card-wrapper {
+  $circle-width: 100px;
+
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+
+  // HACK: use custom CSS variable, set in progress-path component, to set flex alignment
+  // based on the index parity of the card component within progress-path.
+  // Default to center when not a child of progress-path
+  align-items: var(--progress-path-flex-align, center);
```

Comment:
It would be nicer if the variable could just pass the index value (or its parity) to be consumed in the CSS, however applying conditional rules based on a CSS custom property is tricky. There is a way to do it via `@container`, but this doesn't have full browser support: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_containment/Container_size_and_style_queries#container_style_queries